data/language/en-GB.txt

```diff
@@ -1188,7 +1188,7 @@ STR_1813    :Miscellaneous Objects
 STR_1814    :Actions
 STR_1815    :Thoughts
 STR_1816    :{SMALLFONT}{BLACK}Select information type to show in guest list
-STR_1817    :({COMMA16})
+STR_1817    :({COMMA32})
```

Comment:
Should we pursue these changes in `Localisation` ourselves? Seems unnecessary to involve translators if there is nothing to translate

Reply:
Sure. Do note that we still have to change it in the `Localisation` repository and not here.

Reply:
Yes yes, I was just raising the point